query/src/main/scala/filodb/query/exec/MetadataExecPlan.scala

```diff
@@ -120,8 +120,8 @@ final case class PartKeysExec(queryContext: QueryContext,
     }
     Kamon.currentSpan().mark("creating-resultschema")
     val sch = new ResultSchema(Seq(ColumnInfo("TimeSeries", ColumnType.BinaryRecordColumn),
-      ColumnInfo("StartTime", ColumnType.LongColumn),
-      ColumnInfo("EndTime", ColumnType.LongColumn)), 3,
+      ColumnInfo("_startTime_", ColumnType.LongColumn),
+      ColumnInfo("_endTime_", ColumnType.LongColumn)), 3,
```

Comment:
Same goes here. Something like `_partEndTime_`, since we already have `_partIds_`